### ISSUE TITLE
Vagrant Box fails

### ISSUE THREAD

**Issue by unilobo** [mitchellh/vagrant#8371]
Today I have tried to set up a new vagrant box using vdd, something I did only yesterday, and I am now getting the following error:

default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /var/www => C:/VMs/vdd/data
Vagrant was unable to mount VirtualBox shared folders. This is usually
because the filesystem "vboxsf" is not available. This filesystem is
made available via the VirtualBox Guest Additions and kernel module.
Please verify that these guest additions are properly installed in the
guest. This is not a bug in Vagrant and is usually caused by a faulty
Vagrant box. For context, the command attempted was:

mount -t vboxsf -o uid=1000,gid=1000 var_www /var/www

The error output from the command was:

: No such file or directory

### Vagrant version
1.9.2

### Host operating system
Windoze 10

### Guest operating system
Ubuntu

### Vagrantfile
Vagrant.configure("2") do |config|
  # Load config JSON.
  config_json = JSON.parse(File.read("config.json"))

  # Prepare base box.
  config.vm.box = "ubuntu/trusty64"
  config.vm.box_url = "https://atlas.hashicorp.com/ubuntu/boxes/trusty64"

  # Configure networking.
  config.vm.network :private_network, ip: config_json["vm"]["ip"]
  # Configure forwarded ports.
  config.vm.network "forwarded_port", guest: 35729, host: 35729, protocol: "tcp", auto_correct: true
  config.vm.network "forwarded_port", guest: 8983, host: 8983, protocol: "tcp", auto_correct: true
  # User defined forwarded ports.
  config_json["vm"]["forwarded_ports"].each do |port|
    config.vm.network "forwarded_port", guest: port["guest_port"],
      host: port["host_port"], protocol: port["protocol"], auto_correct: true
  end
  # Customize provider.
  config.vm.provider :virtualbox do |vb|
    # RAM.
    vb.customize ["modifyvm", :id, "--memory", config_json["vm"]["memory"]]

    # Synced Folders.
    config_json["vm"]["synced_folders"].each do |folder|
      case folder["type"]
      when "nfs"
        config.vm.synced_folder folder["host_path"], folder["guest_path"], type: "nfs"
        # This uses uid and gid of the user that started vagrant.
        config.nfs.map_uid = Process.uid
        config.nfs.map_gid = Process.gid
      else
        config.vm.synced_folder folder["host_path"], folder["guest_path"]
      end
    end
  end

  # Enable vagrant-hostsupdater support, if the plugin is installed
  # see https://github.com/cogitatio/vagrant-hostsupdater for details
  if Vagrant.has_plugin?("vagrant-hostsupdater")
    config.vm.hostname = "vdd.dev"
    config.hostsupdater.aliases = []

    config_json["vdd"]["sites"].each do |index, site|
        config.hostsupdater.aliases.push(site["vhost"]["url"])
        if site["vhost"]["alias"]
          site["vhost"]["alias"].each do |alias_url|
            config.hostsupdater.aliases.push(alias_url)
          end
        end
    end
  end

  # Run initial shell script.
  config.vm.provision :shell, :path => "chef/shell/initial.sh"
  config.ssh.forward_agent = true
  config.vm.boot_timeout = 120

  config.vm.box_download_insecure
  # Customize provisioner.
  config.vm.provision :chef_solo do |chef|
    chef.json = config_json
    chef.custom_config_path = "chef/solo.rb"
    chef.data_bags_path = "chef/data_bags"
    chef.roles_path = "chef/roles"
    chef.add_role "vdd"
  end
  # Run final shell script.
  config.vm.provision :shell, :path => "chef/shell/final.sh", :args => config_json["vm"]["ip"]

**Comment by vadviktor** [comment]
I am having a recent issue with shared folders too on the same host version.
My Virtualbox is v5.1.16.
Since I am building my boxes with Packer I thought the problem was in the build process (which had not changed for months).
Then I added a new folder manually and it mounted ok.

![image 1](https://cloud.githubusercontent.com/assets/238703/23948812/a1677a9a-097c-11e7-9195-2c45b9d9ebc2.jpg)

The second folder is what I added manually and I could mount it no sweat with `mount -t vboxfs dl /vagrant`.

My Vagrantfile is as simeple as:

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "xenial"
  config.ssh.private_key_path = 'vagrant_rsa'

  config.vm.provider 'virtualbox' do |v|
    v.memory = 1 * 512
    v.cpus   = 1
  end

  config.vm.network 'private_network', ip: '10.0.42.254'
end
```

**Comment by vadviktor** [comment]
Looks like it is related to this issue #8359 and as @rikertothebridge pointed out, Virtualbox broke it in 5.1.16 and will (probably) fix it again in 5.1.17.

I myself am going to downgrade for now back to 5.1.14.

**Comment by unilobo** [comment]
Downgrading to VirtualBox 5.1.14 worked for me d:-)

**Comment by chrisroberts** [comment]
Closing as a dup of #8352

**Comment by vadviktor** [comment]
As a post note, this issue is fixed in the new Virtualbox version 5.1.18.
